dev/tests/unit/Util/ObjectHandlerUtil.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace tests\unit\Util;
+
+use AspectMock\Test as AspectMock;
+use Magento\FunctionalTestingFramework\DataGenerator\Handlers\OperationDefinitionObjectHandler;
+use Magento\FunctionalTestingFramework\DataGenerator\Parsers\OperationDefinitionParser;
+use Magento\FunctionalTestingFramework\ObjectManager;
+use Magento\FunctionalTestingFramework\ObjectManagerFactory;
+
+class ObjectHandlerUtil
+{
+    /**
+     * @var ObjectHandlerUtil
+     */
+    private static $instance;
+
+    /**
+     * ObjectHandlerUtil constructor.
+     */
+    private function __construct()
+    {
+        // private constructor
+    }
+    /**
+     * Static singleton get function
+     *
+     * @return ObjectHandlerUtil
+     */
+    public static function getInstance()
```

Comment:
See what you're trying to accomplish with this class, and I think it's a good idea, but I don't think we should introduce a new singleton. Since this class doesn't keep track of any data, we should refrain from introducing a singleton.

This class should instead be a static class instead with static functions `setMock<type>HandlerWithData($data)`.just like you have it, with a cleanup helper function as well. I confirmed that it works locally in your new testcases.

Reply:
fixed 👍